### DISCUSSION TITLE
Version that supports Python3.9

### DISCUSSION THREAD

**Discussion by OnerSufri** [kedro-org/kedro#739] [Idea]
Hello,

I am wondering if you all have plans to release a version that will support Python 3.9. Thank you

**Comment by datajoely** [comment]
Hi @OnerSufri  - we're hoping to release 3.9 support in either the next minor release 0.17.3 (a couple of weeks from now) or 0.18.0, the next major release (by end of Q2)

**Comment by antonymilne** [comment]
This is now in develop (see 6f9e7e73af5438ff14545523c84657fd49a1006c) so will be in 0.18 🙂

**Comment by datajoely** [comment]
Hey @OnerSufri a little later than hoped - but 3.9 and 3.10 support is now out in 0.18.0!
